@here/harp-mapview/lib/Tile.ts

```diff
@@ -1508,6 +1513,66 @@ export class Tile implements CachedResource {
                         objects.push(outlineObj);
                     }
                 }
+
+                // Add the fill area edges as a separate geometry.
+                if (hasSolidLinesOutlines) {
+                    const outlineTechnique = technique as SolidLineTechnique;
+                    const outlineMaterial = material.clone() as SolidLineMaterial;
+                    const outlineColor = ColorCache.instance.getColor(
+                        getOptionValue(
+                            getPropertyValue(
+                                outlineTechnique.outlineColor,
+                                Math.floor(this.mapView.zoomLevel)
+                            ),
+                            "#000000"
+                        )
+                    );
+                    outlineMaterial.uniforms.diffuse.value = outlineColor;
+                    const outlineObj = new ObjectCtor(bufferGeometry, outlineMaterial);
+
+                    if (outlineTechnique.outlineRenderOrder !== undefined) {
+                        outlineObj.renderOrder = outlineTechnique.outlineRenderOrder;
+                    } else {
+                        if (technique._renderOrderAuto === undefined) {
+                            throw new Error("Technique has no renderOrderAuto");
+                        }
+                        outlineObj.renderOrder = technique._renderOrderAuto - 0.1;
```

Comment:
Yeah, it's questionable at at least in themes out there in wild, we are already defined values at magnitude of 0.1.
Make it smaller, like 0.00001 or something that is literally impractical for user.

I know it's controversial, but as for now only solution. In _future_, we should define uprecision of user supplied renderOrder so we can do hacks like this.

Reply:
Will change this.